package kapsalon.nl.exceptions;

public class ErrorMessages {
    public static final String DIENST_NOT_FOUND = "Dienst met ID ";
    public static final String DIENST_ALREADY_EXISTS = "Dienst met dezelfde kenmerken bestaat al";
    public static final String BARBER_NOT_FOUND = "Barber met ID ";
    public static final String BARBER_ALREADY_EXISTS = "Barber met dezelfde kenmerken bestaat al";

    public static final String KAPSALON_NOT_FOUND = "Kapsalon met ID ";
    public static final String KAPSALON_ALREADY_EXISTS = "Kapsalon met dezelfde kenmerken bestaat al";

    public static final String APPOINTMENT_NOT_FOUND = "Appointment met ID ";
    public static final String APPOINTMENT_ALREADY_EXISTS = "Appointment met dezelfde kenmerken bestaat al";

    public static final String USER_NOT_FOUND = "User met NAME ";
    public static final String USER_ALREADY_EXISTS = "User met dezelfde kenmerken bestaat al";

}
